frontend/src/components/elements/Metric/Metric.tsx

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ * Copyright 2018-2021 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ReactElement } from "react"
+import { Metric as MetricProto } from "src/autogen/proto"
+import ErrorElement from "src/components/shared/ErrorElement"
+import { StyledText, StyledText2, DeltaText } from "./styled-components"
+
+export interface MetricProps {
+  element: MetricProto
+}
+
+export default function Metric({ element }: MetricProps): ReactElement {
+  let direction = ""
+  let color = ""
+  const stRed = "#ff4b4b"
+  const stGreen = "#09ab3b"
```

Comment:
Two things here:

* I think you should be able to pull the `theme` object here (with emotion's `withTheme`) and use a preset color like `green50` and `red50` (I don't know which colors you should actually use here) instead of having the hex codes defined here
* I also think that `theme.colors.red` and `theme.colors.green` define default red/green colors for the theme (which happen to be the same in both light and dark mode, but in general some of these colors do change depending on the theme), and those might be reasonable to use. 

If these colors were picked specifically for metrics by product, then we may want to add them to our big file of predefined colors so that everything is in one place